src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('Since %s I fed them to %s sharks for %.2f seconds' % (z, x, y))
```

Comment:
lol that's a little dark.

You can use \"%s\" to keep the quotes in the I like turtles string.